app/workers/generate_test_applications_for_courses.rb

```diff
@@ -1,15 +1,13 @@
 class GenerateTestApplicationsForCourses
   include Sidekiq::Worker
 
-  def perform(course_ids, courses_per_application, count)
+  def perform(course_ids, courses_per_application)
```

Comment:
while testing locally old jobs errored due to the wrong number of arguments being passed in, we need to watch out for this when deploying this and any existing jobs that could be enqueued

Reply:
Or we can make this backward compatible for now, and remove later

Reply:
That's a really good spot, backwards compatibility sounds sensible! We can leave in the old method verbatim and have a separate PR removing it later